src/test/java/HomeWork19.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import java.time.Duration;
+
+public class HomeWork19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        /*openLoginUrl();*/
+        provideEmail("vera1077@gmail.com");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        selectPlaylist();
+        clickDeletePlaylist();
```

Comment:
Please,    **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**

One test - one Assert
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Reply:
Thank you for the review, I've updated my Homework